apps/samourai-server/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  db:
+    image: "mariadb:10.5.8"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 5m
+    user: "1000:1000"
+    environment:
+        MYSQL_DATABASE: samourai-main
+        MYSQL_ROOT_PASSWORD: rootpassword
+        MYSQL_USER: samourai
+        MYSQL_PASSWORD: password
+    volumes:
+      - ${APP_DATA_DIR}/mysql/data:/var/lib/mysql
+      - ${APP_DATA_DIR}/mysql/db-scripts:/docker-entrypoint-initdb.d
+      - ${APP_DATA_DIR}/mysql/mysql-dojo.cnf:/etc/mysql/conf.d/mysql-dojo.cnf
+      - ${APP_DATA_DIR}/mysql/update-db.sh:/update-db.sh
+    networks:
+       default:
+         ipv4_address: $APP_SAMOURAI_SERVER_DB_IP
+
+  node:
+    image: "louneskmt/dojo-nodejs:1.8.0"
+    logging: *default-logging
+    restart: on-failure
+    command: "/home/node/app/wait-for-it.sh db:3306 --timeout=720 --strict -- /home/node/app/restart.sh"
+    user: "1000:1000"
+    environment:
+      # GLOBAL
+      COMMON_BTC_NETWORK: $BITCOIN_NETWORK
+      DOJO_NODEJS_VERSION_TAG: 1.8.0
+      TOR_PROXY_IP: $TOR_PROXY_IP
+      TOR_PROXY_PORT: $TOR_PROXY_PORT
+
+      # MYSQL
+      MYSQL_DATABASE: samourai-main
+      MYSQL_USER: samourai
+      MYSQL_PASSWORD: password
+
+      # NODEJS
+      NODE_GAP_EXTERNAL: 100
+      NODE_GAP_INTERNAL: 100
+      NODE_ADDR_FILTER_THRESHOLD: 1000
+      NODE_URL_OXT_API: https://api.oxt.me
+      NODE_URL_ESPLORA_API: https://blockstream.info/testnet
```

Comment:
This is hardcoded to testnet.

Also do we even need these options set? Isn't this and `NODE_URL_OXT_API ` only used in the absence of an indexer like electrs?

Reply:
> This is hardcoded to testnet.

That's normal, OXT is for main net and Explore for testnet.

But anyways, I just checked, we won't need any of these two as we use electrs al local indexer.

Reply:
Resolved in fa2c985.